### PR TITLE
Clean up kube-controller-manager.

### PR DESCRIPTION
**Pull request by bjhaid**
These are based on recommendation from
[staticcheck](http://staticcheck.io/).

/kind cleanup

```release-note
NONE
```

### CODE REVIEW

File: cmd/kube-controller-manager/app/cloudproviders.go [hunk 1]
```diff
@@ -48,8 +48,8 @@ func createCloudProvider(cloudProvider string, externalCloudVolumePlugin string,
 		return nil, loopMode, fmt.Errorf("cloud provider could not be initialized: %v", err)
 	}
 
-	if cloud != nil && cloud.HasClusterID() == false {
-		if allowUntaggedCloud == true {
+	if !cloud.HasClusterID() {
+		if allowUntaggedCloud {
 			klog.Warning("detected a cluster without a ClusterID.  A ClusterID will be required in the future.  Please tag your cluster to avoid any future issues")
```
Comment (stewart-yu) [inline cmd/kube-controller-manager/app/cloudproviders.go]: why discard  condition` cloud != nil` ?

Comment (bjhaid) [reply cmd/kube-controller-manager/app/cloudproviders.go]: thanks for catching this, fixed

**Comment by bjhaid** [comment]
@cheftako can you `/ok-to-test` this ❤️

**Comment by xmudrii** [comment]
Hello! We have started the code freeze for 1.15. Is this PR still planned for the 1.15 cycle?

**Comment by bjhaid** [comment]
@cheftako can you approve, thanks!

**Comment by cheftako** [comment]
Hi @bjhaid Sorry only deads2k, lavalamp, mikedanese and sttts can approve this. (See the approvers section of https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-controller-manager/OWNERS)
It looks like sttts and deads2k are already assigned. OTOH its code freeze day and I'm sure they are swamped with requests. You can try sending deads2k a nice note.

**Comment by bjhaid** [comment]
thanks!

**Comment by xmudrii** [comment]
@dims Is this PR needed for 1.15?
@deads2k @sttts @lavalamp @mikedanese Can you please take a look at this PR? Considering this is a small PR and it already had `lgtm`, we can maybe merge it.

**Comment by dims** [comment]
@xmudrii it's good, but not a blocker per se. it's tiny enough and just needs a quick approval :)

**Pull request merged**